Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerSuggestionsViewController.swift

```diff
@@ -188,11 +219,19 @@ class SuggestionsMentionDataSource<ExtraData: ExtraDataTypes>: NSObject,
 
     // MARK: - CollectionViewDataSource
 
-    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    public func collectionView(
+        _ collectionView: UICollectionView,
+        viewForSupplementaryElementOfKind kind: String,
+        at indexPath: IndexPath
+    ) -> UICollectionReusableView {
+        UICollectionReusableView()
+    }
```

Comment:
You probably forgot something?

Reply:
Is it written somewhere in guidelines? I think in the function below, I don't use it :D Anyway I inserted `return` :)